src/wp-includes/fonts/fonts.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Font Library initialization.
+ *
+ * @package    WordPress
+ * @subpackage Fonts
+ * @since      6.5.0
+ */
+
+/**
+ * Registers a new Font Collection in the Font Library.
+ *
+ * @since 6.5.0
+ *
+ * @param string       $slug Font collection slug. May only contain alphanumeric characters, dashes,
+ *                     and underscores. See sanitize_title().
+ * @param array|string $data_or_file {
+ *     Font collection data array or a path/URL to a JSON file containing the font collection.
+ *
+ *     @link https://schemas.wp.org/trunk/font-collection.json
+ *
+ *     @type string $name           Required. Name of the font collection shown in the Font Library.
+ *     @type string $description    Optional. A short descriptive summary of the font collection. Default empty.
+ *     @type array  $font_families  Required. Array of font family definitions that are in the collection.
+ *     @type array  $categories     Optional. Array of categories, each with a name and slug, that are used by the
+ *                                  fonts in the collection. Default empty.
+ * }
+ * @return WP_Font_Collection|WP_Error A font collection if it was registered
+ *                                     successfully, or WP_Error object on failure.
+ */
+function wp_register_font_collection( $slug, $data_or_file ) {
```

Comment:
The code in this file needs to be relocated to `src/wp-includes/fonts.php`, thus removing this file from the `/fonts/` directory.